docs/other/schedd-filebeats.md

```diff
@@ -0,0 +1,110 @@
+Installation of FileBeats for Submit nodes
+==========================================
+
+This document is for frontend administrators. It describes the installation of [Filebeats](https://www.elastic.co/products/beats/filebeat) to continusly upload the HTCondor submit host transfer log to Elastic Search.
+
+
+Introduction
+=================
+
+A submit host (HTCondor schedd) is a login node where users submit jobs to the Grid. One interesting log that it produces is the TransferLog. The TransferLogs report all the transfers of files between compute node sand submit nodes. In this guide we describe the installation of Filebeats to upload this log to Elastic Search.
```

Comment:
> between compute **node sand** submit

typo

Reply:
@brianhlin,

1. I will take a look at the typos. Srry
2. Well I was originally inclined to have it separate since this is kinda a technology preview. The reason behind the documentation is that we would like to get the data from more submit hosts. So instead of me repeating the same email 10 times, I imagined it would be easier just to have it documented. It is by no means needed to run a frontend or not